<?xml version="1.0"?>
<launch>

  <arg name="scan_topic" default="/scan"/>
  <arg name="rosbag" />

  <param name="/use_sim_time" value="true"/>


  <node pkg="iris_lama_ros" name="pf_slam2d_ros" type="pf_slam2d_ros" output="screen" required="true">
    <param name="scan_topic"   value="$(arg scan_topic)" />
    <param name="rosbag" value="$(arg rosbag)" />
  </node>
</launch>
